src/ctelnet.cpp

```diff
@@ -128,7 +128,6 @@ cTelnet::cTelnet(Host* pH, const QString& profileName)
 
     // initialize the socket after the Host initialisation is complete so we can access mSslTsl
     QTimer::singleShot(0, this, [this]() {
-        qDebug() << mProfileName;
```

Comment:
Given that this is a chunk that runs as soon as the main thread becomes idle it might be better to, instead of deleting the debug output, extend the text to indicate more that just the Profile name, e.g. to something like:
```
    qDebug().nospace().noquote() << "cTelnet::cTelnet() INFO - finalizing class network signal/slot connections for profile: \"" << mProfileName << "\".";
```